### PR TITLE
3.1.23

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -18,8 +18,11 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
-3.1.22 (in development)
+3.1.23 (in development)
 -----------------------
+
+3.1.22 - 09/18/22
+-----------------
```
Comment (dschuff) [inline ChangeLog.md:25]: I don't see the changelog entry about C++17 here, is this branch behind?

Comment (sbc100) [reply ChangeLog.md:25]: Oops, fixed.

**Comment by dschuff** [comment]
The PR looks fine. I think the commit log would make more sense with the title we've been using before (i.e. "mark 3.1.22 as released")

**Comment by sbc100** [comment]
Drop in favor of #17886

**Pull request closed without merging**